projects/igniteui-angular/src/lib/combo/combo.common.ts

```diff
@@ -1253,14 +1256,15 @@ export abstract class IgxComboBaseDirective implements IgxComboBase, AfterViewCh
     public onBlur() {
         if (this.collapsed) {
             this._onTouchedCallback();
-            if (this.ngControl && this.ngControl.invalid) {
-                this.valid = IgxInputState.INVALID;
-            } else {
-                this.valid = IgxInputState.INITIAL;
-            }
+            this.validateComboState();
         }
     }
 
+    /** @hidden @internal */
+    public onFocus(): void {
+        this._onTouchedCallback();
+    }
```

Comment:
The actual issue from what I can tell is touched isn't fired on blur when clicking outside - which is fair enough, and that's likely in the close handling - there's a distinction between closing with selection/KB, which restores the focus back to the input (thus focus as a whole has not _yet_ left the component) and outside click which does mean the component is 'blurred' and should use the touched callback.

Reply:
See pretty similar handling in the Date Picker for example that does just that and the else case can be replicated for this fix:
https://github.com/IgniteUI/igniteui-angular/blob/d5df92a8bd090bcad59dab7c2aca6276c2d53d5f/projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts#L927-L933

Reply:
@damyanpetev 
Adding some more details:
_The control cannot be in a touched state on the first focus, that will cause initially invalid fields to turn red on focus, which is not intended at all._
I was thinking the same, yet after the currently implemented fix the red color is updated on the outside click as it should.
![BUG 14900 IG Combo Touched onFocus](https://github.com/user-attachments/assets/180e9fc2-dace-4d6e-b1f4-46d975c95b55)

Reply:
Possibly, because the error color is applied either if the underlying form control changes state (but it's invalid to begin with) or the check we evaluate on blur. Perhaps wasn't the best example, I suppose a better one would be if you have error message based on `model.error.required && model.touched` it should be visible before that :)

On second thought, I believe this will also mess with the `updateOn: 'blur'` on form controls since that pretty mu relies on the touched callback to initiate the value update. I think we had a bunch of issues for that some time ago that showed us we weren't implementing the value accessor right and I thought we hunted those down across all components.

Reply:
Oh more fun, the combo had the same fix as the select... except we reverted that one quick enough lol 😁
See https://github.com/IgniteUI/igniteui-angular/pull/8123

Reply:
@damyanpetev @IvayloG 
I've created a new [PR](https://github.com/IgniteUI/igniteui-angular/pull/14963) to address the issues that this PR introduced, and I'd appreciate your feedback on the proposed solution.

In the new PR I've made a change to the `handleClosing` event:
```typescript
    if (!e.event) {
        this.comboInput?.nativeElement.focus();
    } else if (this.comboInput?.nativeElement !== this.document.activeElement) {
        this._onTouchedCallback();
        this.updateValidity();
    }
```
I've added an else if condition to check if the currently active element is not `this.comboInput`. Initially, I expected this to work with a simple else, but the touched and validation were still somehow being updated unnecessarily. This extra check for the active element seems to prevent those unnecessary updates effectively.

I am sticking to what's written in the Angular's docs => `When the user blurs the form control element, the control is marked as "touched"`. 

Let me know if you have any suggestions or concerns about this approach!